info/release-notes/24.next.0-readme.txt

```diff
@@ -60,7 +60,8 @@ Changes
 Bugfix
 ======
 
-* 
+* TCX Import (Frederic)
+  - Fix for average and maximum power computation
```

Comment:
Can you add the link to this issue

Reply:
Fixed in [this PR](https://github.com/mytourbook/mytourbook/pull/1288)